.cargo/musl-gcc

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Wrapper for building with musl.
+#
+# musl unfortunately requires a musl-enabled C compiler (musl-gcc) to be
+# installed, which can be kind of a pain to get installed depending on the
+# distro. That's not a very good user experience.
+#
+# The real musl-gcc wrapper sets the correct system include paths for linking
+# with musl libc library. Since this is not actually used to link any binaries
+# it should most likely work just fine.
+
+gcc "$@"
```

Comment:
I don't think we need those extra wrappers, do we? We're only calling gcc/g++ with no extra flags

Reply:
We do, compiling with the `x86_64-unknown-linux-musl` target will fail otherwise.

Reply:
isn't it enough to set `CC_x86_64_unknown_linux_musl = { value = "gcc", force = true, relative = true }` ?

Reply:
Indeed, looks like that works (without `relative = true`), though we lose the explanatory comments in the files. For that reason I'd rather keep the `musl-gcc` files.